src/System.Net.Sockets/ref/4.1.0/System.Net.Sockets.depproj

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
In general things should be harvested and we shouldn't need to add any depproj's. The only cases where we have needed to do it is for things like picking the older contracts for APICompat checks which sound like what you are hitting here. The other option we have is to just disable APICompat checks for the older builds, see https://github.com/dotnet/corefx/blob/71de1aac5cf3e5dc87714ae0cc829e1fa98f7c5d/src/System.IO.FileSystem/src/System.IO.FileSystem.csproj#L16.